Added type hints to ResNetForImageClassification

kishore-s-15 opened the pull request:
Based on Issue #16059.

While looking into Issue #16059, I found that the `ResNetForImageClassification` model's type hints are inconsistent with the [docs](https://huggingface.co/docs/transformers/model_doc/resnet#transformers.ResNetForImageClassification.forward). Modified it to be consistent with the docs.

@Rocketknight1 Could you kindly check if this is fine?

This is my First PR for the Transformers library. Thanks in advance.

kishore-s-15 committed:
Added type hints to ResNetForImageClassification

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_19084). All of your documentation changes will be reflected on that endpoint.

kishore-s-15 committed:
Resolved check_repository_consistency failure issue

Running fix-copies changed the type hints for RegNetForImageClassification in modeling_regnet.py file

Rocketknight1 reviewed (approved):
LGTM, thank you!

kishore-s-15 commented:
Thanks @Rocketknight1.
